packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js

```diff
@@ -272,6 +274,7 @@ export default class IpListController {
           $scope.ipsList = map(ips, (ip) => {
             return {
               ...ip,
+              ip: ip.ip || ip.ipBlock,
```

Comment:
`ip.ipBlock` is set with `ip.ip` so it's useless

Reply:
ip.ipBlock returned by API and in my case i need to set ip field because ipv6 in the fact is not ipBlock